src/launcher/standalone_launcher/package.json

```diff
@@ -0,0 +1,166 @@
+{
+  "name": "standalone-launcher",
+  "description": "Standalone Launcher, laucher of reactive apps.",
```

Comment:
Typo: s/laucher/launcher

Reply:
chore: PR Clean UP(ARTP1353)

Remove the redundant codes
Remove the unused dependencies
Flatten the directory
Delete the docker and ngnix related directories.